### ISSUE TITLE
Player Scores and Stats in general no longer possible

### ISSUE THREAD

**Issue by kodarfour** [dtsong/sleeper-api-wrapper#18]
Wanted to bring up the fact that the Stats() functions and all others that use it no longer operate because sleeper removed it from the API.

**Comment by wfordh** [comment]
Yeah I've noticed that too. It actually is still supported in the API - I've made the calls and it works - but I am working to clean up that file and at least add a warning to the user that we can't guarantee the results / support continuing. Deprecation is an option, too.
